hook/gomod/gomod.go

```diff
@@ -0,0 +1,92 @@
+package gomod
+
+import (
+	"path/filepath"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/aquasecurity/fanal/hook"
+	"github.com/aquasecurity/fanal/types"
+)
+
+func init() {
+	hook.RegisterHook(gomodMergeHook{})
+}
+
+const version = 1
+
+type gomodMergeHook struct{}
+
+// Hook removes files installed by OS package manager such as yum.
```

Comment:
This comment doesn't reflect what the function does.

Reply:
Nice catch. Thanks.